src/main/resources/application.yml

```diff
@@ -18,7 +20,39 @@ zeebe:
   broker:
     contactpoint: "127.0.0.1:26500"
 
+#ams:
+#  local:
+#    server-cert-check: false
+#    enabled: true
+
 ams:
   local:
     server-cert-check: false
-    enabled: false
+    enabled: true
+    interop:
+      host: http://rhino.sandbox.fynarfin.io
+    customer:
+      host: http://rhino.sandbox.fynarfin.io
+    account:
+      host: http://rhino.sandbox.fynarfin.io
+    auth:
+      host: http://rhino.sandbox.fynarfin.io
+    tenants:
+      - name: "gorilla"
+        user: mifos
+        password: password
+        authtype: basic
+        fspId: in02tn03
+      - name: "lion"
+        user: mifos
+        password: password
+        authtype: basic
+        fspId: in02tn04
+
+
+# Savings : S , Loan : L
+accountPrefixes: "S,L"
```

Comment:
This should be an array as per design

Reply:
This is an array. Reference : https://stackoverflow.com/questions/26699385/spring-boot-yaml-configuration-for-a-list-of-strings